dev/CommonStyles/InkToolbar_themeresources.xaml

```diff
@@ -1279,7 +1580,7 @@
                             <VisualStateGroup x:Name="GlyphVisualStates">
                                 <VisualState x:Name="ShowExtensionGlyph">
                                     <VisualState.Setters>
-                                        <Setter Target="ExtensionGlyph.Opacity" Value="1" />
+                                        <Setter Target="ExtensionGlyph.Opacity" Value="0" />
```

Comment:
Same here, this behavior is too tricky. We should remove all ExtensionGlyph related code.